cmd/opm/registry/prunestranded.go

```diff
@@ -0,0 +1,54 @@
+package registry
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/lib/registry"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newRegistryPruneStrandedCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "prune-stranded",
+		Short: "prune an operator registry DB of stranded bundles",
```

Comment:
nit: I believe docker uses the word `dangling` for this, which might help users understand the issue. But doesn't need to change.